iot/http_example/cloudiot_http_example.js

```diff
@@ -166,6 +166,32 @@ function publishAsync (messageCount, numMessages) {
 }
 // [END iot_http_publish]
 
+// [START iot_http_getconfig]
+function getConfig (version) {
+  console.log('Getting config:');
+  console.log(urlBase);
+  const options = {
+    url: urlBase + '/config?local_version=' + version,
+    headers: {
+      'authorization': 'Bearer ' + authToken,
```

Comment:
Nit: use template strings instead of concatenation? e.g:
```
`Bearer ${authToken}`
```

If you decide to make this change, please update the rest of your PR as well.